htlc/src/components/createEscrow.js

```diff
@@ -0,0 +1,9 @@
+// const algob = require("@algorand-builder/algob");
+
+async function CreateEscrow() {
+    const sndrAddr = prompt("Enter Sender Address");
+    const recvAddr = prompt("Enter Receiver Address");
```

Comment:
Can we use a form instead of prompt?

Reply:
Also, how is this creating an escrow account?

Reply:
This file will be removed. I am very new to react, i.e. used simple prompts for getting the data from the user.